package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "/usr/local/bin/codeclimate",
```

Comment:
@Arcanemagus Do you mean something like:

```javascript
        let result;
        try {
          result = await Helpers.exec('/bin/bash', execArgs, execOpts);
        } catch (e) {
          if (e.code === 'ENOENT' && !badPaths.has(this.executablePath)) {
            const msg = 'codeclimate binary not found! Installation ' +
              'instructions at http://github.com/codeclimate/codeclimate';
            atom.notifications.addError(msg);
            // Only notify once per path
            badPaths.add(this.executablePath);
            return [];
          }
        }
```
instead [this](https://github.com/AtomLinter/linter-codeclimate/blob/d0c42f46717d19c2ed61b868eef0e676c3a4ebe5/lib/index.js#L173) and removing [this block](https://github.com/AtomLinter/linter-codeclimate/blob/d0c42f46717d19c2ed61b868eef0e676c3a4ebe5/lib/index.js#L78-L88)?

Got it working for both executablePath, `codeclimate` and `/usr/bin/local/codeclimate`.

Let me know what you think. I should also restore the default for `linter-codeclimate.executablePath` and change the description to `The path to the 'codeclimate' executable`.